dev/TabView/TabView.cpp

```diff
@@ -208,6 +208,39 @@ void TabView::OnSelectedItemPropertyChanged(const winrt::DependencyPropertyChang
 void TabView::OnTabWidthModePropertyChanged(const winrt::DependencyPropertyChangedEventArgs&)
 {
     UpdateTabWidths();
+
+    // Switch the visual states of all tab items to compact
```

Comment:
nit: Can you please update this comment -- they're not being change to compact, they're being updated of the new width mode.

Reply:
Fixed